corehq/apps/app_manager/templates/app_manager/partials/enable_schedule.html

```diff
@@ -1,29 +1,83 @@
+                            <div class="alert alert-error">
+                                <strong>
+                                    {% blocktrans %}
+                                        Visit schedule will be ignored as not all
+                                        forms in the module update or close a case.
+                                    {% endblocktrans %}
+                                </strong>
+                            </div>
+                        {% endif %}
+                    </label>
                 </div>
-                {% endif %}
-            {% else %}
-                {% blocktrans %}
-                    Visit schedules can only be used on modules where all the forms in the module<br/>
-                    update or close a case, which means registration forms must go in a separate module.
-                {% endblocktrans %}
-            {% endif %}
-        </label>
+            </div>
+        </form>
+        <fieldset data-bind="visible: hasSchedule">
+            <legend>{% trans "Schedule Phases" %}</legend>
+            <table class="table table-hover table-condensed">
+                <thead>
+                    <th></th>
+                    <th>{% trans "Schedule Phase Number" %}</th>
+                    <th>{% trans "Schedule Anchor Property" %}</th>
+                    <th>{% trans "Forms in this phase" %}</th>
+                    <th></th>
+                </thead>
+                <tbody data-bind="template: { name: 'phase-template', foreach: phases},
+                                  sortableList: phases"></tbody>
+            </table>
+            <button class="btn" data-bind="click: addPhase">{% trans "Add Phase" %}</button>
+        </fieldset>
     </div>
-</div>
+    <script id="phase-template" type="text/html">
+        <tr class="phase" data-bind="style: {cursor: 'move'}">
+            <td>
+                <i class="grip sortable-handle"
+                   data-bind="css: COMMCAREHQ.icons.GRIP,
+                              event: {mousedown: function(){ $(':focus').blur(); }},
+                              visible: $data !== $parent.selectedPhase()"></i>
+                <input data-bind="value: name, visibleAndSelect: $data === $parent.selectedPhase(),
+                                  event: { blur: function() { $parent.selectPhase(''); } }"/>
+            </td>
+            <td data-bind="text: $index() + 1"></td>
+            <td>
+                <div href="#" data-bind="jqueryElement: anchor.ui"></div>
+            </td>
+            <td data-bind="text: form_abbreviations"></td>
+            <td>
+                <i style="cursor: pointer;"
+                   data-bind="css: COMMCAREHQ.icons.DELETE,
+                              click: $parent.removePhase,
+                              visible: forms().length == 0
+                              "></i>
+                <i style="cursor: pointer;"
+                   class="icon-question-sign icon-blue"
```

Comment:
I tried doing that, but it wasn't playing nicely with being able to drag and drop the table rows.